docs/v3/identity.md

```diff
@@ -0,0 +1,261 @@
+# Multi-wallet Identity in V3
+
+## Summary
+
+* XMTP v3 now supports multiple wallets per user, including [smart wallets](https://community.xmtp.org/t/xip-44-smart-contract-wallet-support/627).
+* User wallets now belong to a shared Inbox with a single Inbox ID.
+* The first wallet used to create an Inbox is a *recovery wallet*.
+  * A recovery wallet can remove other wallets.
+  * A user can use the recovery wallet to set a different recovery wallet.
+  * A user with an existing V2 wallet gets an Inbox with that wallet as the recovery wallet.
+* The Inbox can have multiple *associated wallets* (besides the recovery wallet).
+  * Users can send a message to any name belonging to an associated wallet in the Inbox.
+  * Associated wallets can add one or more *installations* as children (one per app/device pairing).
+  * Removing the wallet removes all installations that are the child of that wallet.
+* Adding/removing a wallet or installation is an *identity action*.
+  * Only recovery wallets can remove other wallets or installations.
+* Identity actions for new associations are signed by the wallet or installation making the change.
+  * In some cases, it may be signed by both.
+  * Action removing associations and changing the recovery wallet must be signed by the recovery wallet.
```

Comment:
> * Action removing associations and changing the recovery wallet must be signed by the recovery wallet.

This seems to cover the `Only recovery wallets can remove other wallets or installations.`

Reply:
This bullet has to do more specifically with how signing works.  It is a consequence of the earlier bullet.